### PR TITLE
Add consistency_level in search/query request

### PR DESCRIPTION
**Pull request by czs007**
issue #1464

### CODE REVIEW

File: pymilvus/client/prepare.py [hunk 1]
```diff
@@ -559,14 +463,17 @@ def search_requests_with_expr(cls, collection_name, data, anns_field, param, lim
             "ignore_growing": ignore_growing,
         }
 
+        if anns_field:
+            search_params["anns_field"] = anns_field
+
```
Comment (XuanYang-cn) [inline pymilvus/client/prepare.py:468]: anns_field can be None?

Comment (czs007) [reply pymilvus/client/prepare.py:468]: yes, server will handle this. proxy will deduce it from schema.
see https://github.com/milvus-io/milvus/pull/24499

**Comment by XuanYang-cn** [comment]
LGTM
/assign @longjiquan

**Review by longjiquan — APPROVED** [review]
/lgtm

**Pull request merged**